frontend/src/components/EditAndReviewTable.jsx

```diff
@@ -177,8 +177,9 @@ function StatusTableRow({
   // user role
   const { ROLES, checkAccess } = useAccess();
   const approver = useMemo(
-    () => checkAccess(ROLES.APPROVER),
-    [checkAccess, ROLES.APPROVER],
+    () =>
+      checkAccess([ROLES.APPROVER, ROLES.SUPER_ADMIN, ROLES.ALL_PARK_ACCESS]),
+    [checkAccess, ROLES.APPROVER, ROLES.SUPER_ADMIN, ROLES.ALL_PARK_ACCESS],
```

Comment:
This "approver" may not be appropriate since it has more than that role. Any suggestions for a name?